manuscript/manuscript.Rmd

```diff
@@ -439,17 +439,14 @@ Such examination makes it clear how the RL agent's use of a more dynamic strateg
 The various kinks in the RL policy function also suggest that this solution is nevertheless surely not the optimal solution -- such features are typical of RL-based policies, where such spandral-like features from early experimentation may become baked into the long-term strategy if adjusting them has little improvement on the final reward. <!-- ML: the parenthetical here I understand but I feel a little ehh on the word choice. I also don't know how to rephrase it -->
 Further tuning of hyper-parameters, increased training, alterations or alternatives to the training algorithm would likely be able to further improve upon this performance.
 
-## Unsolved environments: Wildfires and other examples
+## Additional Environments
 
-<!-- ML: Need to rework this to talk about other environments that we are working on basically point to conservation gym website -->
-One feature that most distinguishes RL from other approaches to decision theory (such as optimal control) is it's flexibility or ambition.
-While optimal control approaches typically require extensive simplifying assumptions such as discrete states or actions, deterministic dynamics or short time horizons, RL is scoped more ambitiously.
-Large, complex problems are still immensely challenging to RL, and existing RL approaches will frequently struggle to solve problems that can already be solved much more efficiently by other means.
-Nevertheless, deep RL approaches using neural network architectures have proven successful on problems that no amount of computational power would put into reach of purely conventional methods.
-Our previous examples have focused on relatively simple examples that can also be addressed at least approximately by conventional methods, and also permit RL-based solutions to be found relatively efficiently on commodity hardware.
-To better illustrate the generality of RL approach and more typical complexity of conservation problems, we now introduce a third example for which we have not yet found an effective policy using RL or classical methods.
+Deep RL algorithms have proven successful on a range of problems that classical approaches could not solve despite immense computational resources -- e.g. deep RL can solve control problems with image-based observations that would be intractable with classical methods [@DQN]--; yet, getting deep RL to work in practice can be unexpectedly challenging and computationally intensive.
+Our previous examples have thus focused on relatively simple problems that can be addressed by conventional methods, and also permit RL-based solutions to be found relatively efficiently on commodity hardware.
```

Comment:
I think we will get in trouble about our vague use of the term conventional methods.  I've mostly used this in a very narrow sense of SDP-based methods, which covers most of the actual examples in ecology.  We set up the second example with a non-autonomous change explicitly to rule out a direct SDP solution (technically our tipping point example is a partially-observed system, as you do not observe the value of the time-varying parameter moving the system towards the bifurcation).  I know our examples are still simple, but I think this contradicts our earlier statements about example 2, particularly as we don't have any optimal conventional solution to compare the results to.